tests/examples/opinions/united_states/prapp_example.compare.json

```diff
@@ -0,0 +1,2042 @@
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "Firstbank Puerto Rico v. Hernandez Rios, Hilario",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLAN202200672-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLAN202200672",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "Feliciano Maldonado, Magalis v. Coop De Ahorro Y Credito Yabucoe\u00f1a",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLAN202200320-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLAN202200320",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "El Pueblo De Puerto Rico v. Torres Gonzalez, Juan Orlando",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLAN202200215-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLAN202200215",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "El Pueblo De Puerto Rico v. Gonzalez Caraballo, Obett A",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLAN202200335-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLAN202200335",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "El Pueblo De Puerto Rico v. Danet Mendrell, Louis",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLCE202201192-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLCE202201192",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "El Pueblo De Puerto Rico v. Bonilla Rosado, Jose Carmelo",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLCE202201213-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLCE202201213",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "Dorado Beach Homeowners Association v. Irizarry Santiago, Ernesto R.",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLAN202200529-30112022.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "KLAN202200529",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2022-11-30",
+    "case_names": "Delgado Diaz, Javier v. Junta De Libertad Bajo Palabra",
+    "download_urls": "https://www.poderjudicial.pr/ta/2022/KLRA202200340-30112022.pdf",
```

Comment:
I forget the right answer, but I see that this download_url has the domain and that the other test fixture does not. One is probably right and the other wrong, right? Which needs fixing?

Reply:
Both are correct.  Both are allowed by the system.  I believe is the right answer.

Reply:
trimmed down